sample/lib/main.dart

```diff
@@ -14,21 +15,22 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return BlocProvider<PageBloc>(
-      builder: (context) => PageBloc(),
-      child: MaterialApp(
-      title: 'GTU BT',
-      theme: ThemeData(
-        // These will be customized for general app design
-        primarySwatch: Colors.blue,
-      ),
-      routes: Routes.routes,
-      initialRoute: '/',
-    )
+        builder: (context) => PageBloc(),
+        child: MaterialApp(
+          title: 'GTU BT',
+          theme: ThemeData(
+            // These will be customized for general app design
+            primarySwatch: Colors.blue,
```

Comment:
bu renkleri bu sekilde kullanmayalim. @ahmetergani  bir sinif olusturdu orada tanimlayalim ve oyle kullanalim. 
(dev kanalina @ahmetergani nin degisikligi merge edildi dev branch'ini bu branch'e rebase ettikten sonra bu color'lari degistir.)

Reply:
Tamam, not ettim.